opentelemetry/proto/common/v1/common.proto

```diff
@@ -53,3 +53,10 @@ message StringKeyValue {
   string key = 1;
   string value = 2;
 }
+
+// InstrumentationLibrary is a message representing the instrumentation library informations
+// such as the fully qualified name and version. 
+message InstrumentationLibrary {
+  string name = 1;
+  string version = 2;
```

Comment:
Me might consider using a structured way to represent the version in the future to split up the ` semver:1.2.3` into two parts.

Reply:
Do note that more fields and structure usually means slower performance and more memory allocations with ProtoBufs.
A string with well-defined conventions appears to be the right balance here.